watertap/examples/flowsheets/case_studies/wastewater_resource_recovery/dye_desalination/dye_desalination.py

```diff
@@ -131,6 +226,76 @@ def set_operating_conditions(m):
     dye_sep.P1.eta_pump.fix(0.75)  # pump efficiency [-]
     dye_sep.P1.lift_height.unfix()
 
+    m.fs.tb_nf_gac.properties_out[0].temperature.fix(298.15)
+    m.fs.tb_nf_gac.properties_out[0].pressure.fix(101325)
+
+    m.fs.gac.freund_k.fix(10)
+    m.fs.gac.freund_ninv.fix(0.9)
+    m.fs.gac.shape_correction_factor.fix()
+    m.fs.gac.particle_porosity.fix()
+    m.fs.gac.tort.fix()
+    m.fs.gac.spdfr.fix()
+    # gac particle specifications
+    m.fs.gac.particle_dens_app.fix(750)
+    m.fs.gac.particle_dia.fix(0.001)
+    # adsorber bed specifications
+    m.fs.gac.ebct.fix(600)
+    m.fs.gac.bed_voidage.fix(0.4)
+    m.fs.gac.bed_length.fix(6)
+    # design spec
+    m.fs.gac.conc_ratio_replace.fix(0.50)
+    # parameters
+    m.fs.gac.a0.fix(3.68421)
+    m.fs.gac.a1.fix(13.1579)
+    m.fs.gac.b0.fix(0.784576)
+    m.fs.gac.b1.fix(0.239663)
+    m.fs.gac.b2.fix(0.484422)
+    m.fs.gac.b3.fix(0.003206)
+    m.fs.gac.b4.fix(0.134987)
+
+    m.fs.gac.gac_removed[0].flow_mass_phase_comp["Liq", "H2O"] = 1e-10
+    m.fs.gac.gac_removed[0].flow_mass_phase_comp["Liq", "tds"] = 1e-10
```

Comment:
@hunterbarber We are trying to get this `dye_desalination.py` flowsheet to solve before the `dye_desalination_withRO.py`, but the solver always seem to enter restoration. So far we've tried the following:

- L138: Tweaking the dye diffusivity value (which we don't currently have an actual number for)
- L256-257: Setting the problematic zero-flow terms in the `gac_removed` streams to a small, non-zero value. We had also initially set `ele_operational_time` and `ele_conc_ratio_replace` to zero, but this didn't improve the solver stability at all

Have you encountered such issues with the GAC model before and/or do you have any advice for how we should go about solving this (Variables to update, bounds to relac/tighten, etc.)?